sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -87,6 +87,20 @@ public void testCreateViewFailsIfNameConflictsWithPartitionedTable() {
         execute("create view t1 as select 1");
     }
 
+    @Test
+    public void testCreateTableFailsIfNameConflictsWithView() {
+        execute("create view v4 as select 1");
+        expectedException.expectMessage("Relation '" + sqlExecutor.getDefaultSchema() + ".v4' already exists");
+        execute("create table v4 (x int) clustered into 1 shards with (number_of_replicas = 0)");
```

Comment:
Should we remove the analyzer tests and analyzer handling to make sure that this actually triggers the logic in the new transport action?

Another pattern would be to utilize `plan` (`PlanForNode`) to analyze the statement first, then create the table, and then execute the plan that was created prior to the table existing.

Reply:
I tried updating the ClusterState inside `Schemas` to get rid of the view. Didn't really work because the master pushes state updates which revert the removal again.

Ideally we would like to have two checks, one locally in the analyzer to avoid unnecessary network traffic and one when performing the operation. Arguably the second one is more important, so I'm leaning towards removing the analyzer check if testing is too complicated. I'll try the plan approach.

Reply:
I think we could neglect the impact of the unnecessary network traffic in case of a conflict. Creating tables is rare, that somebody wants to create a table where a view already exists is even rarer.

Reply:
That was actually straight-forward. Thanks for the tip!

Reply:
Didn't see your last comment before I posted. I agree, it's neglectable. However, it is consistent because we have the same check for tables in the analyzer which wouldn't be strictly necessary either.